01_Data/src/layers/sequelize/types/DateOnlyColumn.ts

```diff
@@ -0,0 +1,18 @@
+import {Column, DataType, Model} from "sequelize-typescript";
+import {DateOnly, safelyMap} from "@citrineos/base";
+import {TariffElement} from "../model/Tariff/TariffElement";
+
+// TODO: custom type would be preferable
```

Comment:
I tried registering a custom type in Sequelize for it but couldn't get it to work. I think a custom type would be a more elegant solution. Perhaps we can revisit it when we have more time?